templates/_layouts/docs.html

```diff
@@ -48,145 +48,25 @@
-                {{ side_nav_item("/docs/patterns/table-of-contents", "Table of contents") }}
-                {{ side_nav_item("/docs/patterns/tabs", "Tabs") }}
-                {{ side_nav_item("/docs/patterns/tooltips", "Tooltips") }}
-              </ul>
-
-              <ul class="p-side-navigation__list">
-                <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">Utilities</span></li>
-                {{ side_nav_item("/docs/utilities/align", "Align") }}
-                {{ side_nav_item("/docs/utilities/baseline-grid", "Baseline grid") }}
-                {{ side_nav_item("/docs/utilities/clearfix", "Clearfix") }}
-                {{ side_nav_item("/docs/utilities/embedded-media", "Embedded media") }}
-                {{ side_nav_item("/docs/utilities/equal-height", "Equal height") }}
-                {{ side_nav_item("/docs/utilities/floats", "Floats") }}
-                {{ side_nav_item("/docs/utilities/font-metrics", "Font metrics") }}
-                {{ side_nav_item("/docs/utilities/functions", "Functions") }}
-                {{ side_nav_item("/docs/utilities/hide", "Hide") }}
-                {{ side_nav_item("/docs/utilities/image-position", "Image position") }}
-                {{ side_nav_item("/docs/utilities/margin-collapse", "Margin collapse") }}
-                {{ side_nav_item("/docs/utilities/no-print", "No print") }}
-                {{ side_nav_item("/docs/utilities/off-screen", "Off-screen") }}
-                {{ side_nav_item("/docs/utilities/padding-collapse", "Padding collapse") }}
-                {{ side_nav_item("/docs/utilities/show", "Show") }}
-                {{ side_nav_item("/docs/utilities/table-cell-padding-overlap", "Table cell padding overlap") }}
-                {{ side_nav_item("/docs/utilities/has-icon", "Text with icon") }}
-                {{ side_nav_item("/docs/utilities/truncate", "Truncation") }}
-                {{ side_nav_item("/docs/utilities/vertical-spacing", "Vertical spacing") }}
-                {{ side_nav_item("/docs/utilities/vertically-center", "Vertically center") }}
-              </ul>
-
+              
+              {% for item in sideNavigation %}
               <ul class="p-side-navigation__list">
-                <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">Layouts</span></li>
-                {{ side_nav_item("/docs/layouts/application", "Application") }}
-                {{ side_nav_item("/docs/layouts/documentation", "Documentation") }}
-                {{ side_nav_item("/docs/layouts/fluid-breakout", "Fluid breakout") }}
-                {{ side_nav_item("/docs/layouts/full-width", "Full-width") }}
-                {{ side_nav_item("/docs/layouts/sticky-footer", "Sticky footer") }}
+                <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">{{ item.heading | safe | replace("%version", version) }}</span></li>
```

Comment:
The changes (new format and removal of `safe`) should be applied here too I guess?

Reply:
okay. I don't see anywhere this might be needed atm. So removing it